### PR TITLE
pdn: Sort tcl commands args in documentation.

### PR DESCRIPTION
**Pull request by titan73**

### CODE REVIEW

File: src/pdn/README.md [hunk 1]
```diff
@@ -20,26 +20,26 @@ be defined over the stdcell area, and over areas occupied by macros.
...
+pdngen
     [-dont_add_pins]
+    [-failed_via_report file]
+    [-report_only]
     [-reset]
     [-ripup]
-    [-report_only]
-    [-failed_via_report file]
+    [-skip_trim]
     [-verbose]
```
Comment (maliberty) [inline src/pdn/README.md:30]: -verbose is missing in the list of options below

File: src/pdn/README.md [hunk 2]
```diff
@@ -164,24 +163,24 @@ define_power_switch_cell -name POWER_SWITCH -control SLEEP -switched_power VDD -
...
 | ----- | ----- |
-| `-name` | The name of the power switch cell. |
 | `-control` | The name of the power control port of the power switch cell. |
-| `-switched_power` | Defines the name of the pin that outputs the switched power net. |
-| `-power` | Defines the name of the pin that connects to the unswitched power net. |
 | `-ground` | Defines the name of the pin that connects to the ground net. |
+| `-name` | The name of the power switch cell. |
+| `-power` | Defines the name of the pin that connects to the unswitched power net. |
+| `-switched_power` | Defines the name of the pin that outputs the switched power net. |
 | `-acknowledge` | Defines the name of the output control signal of the power control switch if it has one. |
```
Comment (maliberty) [inline src/pdn/README.md:184]: Do we want to group these in the detailed options list too?  The mandatory/optional distinction isn't clear and it just looks unsorted.  Alternatively use [-acknowledge] to clarify.

File: src/pdn/README.md (hunk 1 above)
Comment (titan73) [reply src/pdn/README.md:30]: Yes I saw it. I saw other issues too. I'll fix them in another PR. This PR only sorts args.

File: src/pdn/README.md (hunk 2 above)
Comment (titan73) [reply src/pdn/README.md:184]: With grouping it matches the usage and it's easier to see discrepancies and missing option description. I don't have a strong opinion on that though. What do you think?

Comment (titan73) [reply src/pdn/README.md]: I've updated the PR with the brackets on optional flags in description.

Comment (titan73) [reply src/pdn/README.md]: @gadfort I'll fixed the issues in pdn doc. Here are the ones I found;

pdngen: -verbose missing from description. I'll found one.

define_pdn_grid: -power_control_signal in usage and -power_control in description. I guess the usage one is correct

define_power_switch_cell: -power_switchable in usage and -switched_power. I guess the usage is correct too.

adn_pdn_ring: -ground_pads and -power_pads missing from description. What description would you add?

File: src/pdn/README.md [hunk 3]
```diff
@@ -111,46 +111,45 @@ define_pdn_grid -name main_grid -existing
...
-| `-power_control` | Defines the name of the power control signal used to control the switching of the inserted power switches. |
-| `-power_control_network` | Defines the structure of the power control signal network. Choose from STAR, or DAISY. If STAR is specified, then the network is wired as a high-fanout net with the power control signal driving the power control pin on every power switch. If DAISY is specified then the power switches are connected in a daisy-chain configuration - note, this requires that the power swich defined by the `define_power_switch_cell`  command defines an acknowledge pin for the switch. |
-| `-existing` | Flag to enable defining for existing routing solution. |
+| `[-cells]` | For a macro, defines a set of valid cells. Macros which are instances of one of these cells will use this grid specification. |
+| `[-default]` | For a macro, specifies this is a default grid that can be overwritten. |
+| `[-existing]` | Flag to enable defining for existing routing solution. |
+| `[-grid_over_pg_pins]`, `-grid_over_boundary` | Place the power grid over the power ground pins of the macro. (Default True), or Place the power grid over the entire macro. |
```
Comment (maliberty) [inline src/pdn/README.md:140]: Add [] to -grid_over_boundary too

**Review by maliberty — CHANGES_REQUESTED** [review]

File: src/pdn/README.md (hunk 3 above)
Comment (titan73) [reply src/pdn/README.md]: Fixed. Thanks.

File: src/pdn/README.md (hunk 2 above)
Comment (gadfort) [reply src/pdn/README.md]: -power_control is correct https://github.com/The-OpenROAD-Project/OpenROAD/blob/ea853cdd83be73ec2fe8bcbaebae891773cfca65/src/pdn/src/pdn.tcl#L864
-switched_power is correct https://github.com/The-OpenROAD-Project/OpenROAD/blob/ea853cdd83be73ec2fe8bcbaebae891773cfca65/src/pdn/src/pdn.tcl#L93
add_pdn_ring: -ground_pads and -power_pads these are deprecated and should not be documented.

Comment (titan73) [reply src/pdn/README.md]: > -switched_power is correct
> 
> https://github.com/The-OpenROAD-Project/OpenROAD/blob/ea853cdd83be73ec2fe8bcbaebae891773cfca65/src/pdn/src/pdn.tcl#L93
> 
It was about define_power_switch_cell. I guess -power_switchable is correct then.

sta::define_cmd_args "define_power_switch_cell" {-name <name> \
                                                 -control <control_pin> \
                                                 [-acknowledge <acknowledge_pin>] \
                                                 -power_switchable <power_switchable_pin> \
                                                 -power <power_pin> \
                                                 -ground <ground_pin> }

**Review by maliberty — APPROVED** [review]

**Pull request merged**